client/components/DrawingGame.jsx

```diff
@@ -94,7 +94,8 @@ export default class DrawingGame extends Component {
         <button type="button" onClick={() => handleButton(false, false)}>Stop Drawing</button>
         <button type="button" onClick={() => handleButton(false, true)}>Erase</button>
         <button type="button" onClick={() => drawingContext.clearRect(0, 0, drawingCanvas.width, drawingCanvas.height)}>Clear</button>
-        <ThreeDScene socket={this.state.socket} />
+        {socket.role ? <div>You are the {socket.role}</div> : null}
```

Comment:
don't change the rendering in the parent component based on changes to the game state.  i would just use a console.log if you want to double check the role

Reply:
for sure. I'll make another pull request that fixes this.